source/common/http/async_client_impl.h

```diff
@@ -411,7 +411,7 @@ class AsyncStreamImpl : public AsyncClient::Stream,
   void removeDownstreamWatermarkCallbacks(DownstreamWatermarkCallbacks&) override {}
   void setDecoderBufferLimit(uint32_t) override {}
   uint32_t decoderBufferLimit() override { return 0; }
-  bool recreateStream() override { return false; }
+  bool recreateStream(const ResponseHeaderMap* /*headers*/) override { return false; }
```

Comment:
I think generally we skip spelling out the name when omitted, so I would remove this to be consistent with the rest of the file at the very least

Reply:
Got it, fixed.